mailbag/derivatives/pdf.py

```diff
@@ -0,0 +1,43 @@
+import os
+from subprocess import Popen, PIPE
+from mailbag.derivative import Derivative
+
+
+class ExampleDerivative(Derivative):
+    derivative_name = 'pdf'
+
+    def __init__(self, email_account, **kwargs):
+        print("Setup account")
+        super()
+
+    def do_task_per_account(self):
+        print(self.account.account_data())
+
+    def do_task_per_message(self, message, args):
+        wkhtmltopdf = 'wkhtmltopdf.exe'
+        html = message.HTML_Body
+        From_name=message.From.split("<")
```

Comment:
Looks like this is usually failing for me on these split methods. First, don't worry about parsing the text, just write it. We have to handle all sorts of strings and parsing adds a lot of risk and having outputs with brackets is fine. That's also how the addresses are often stored in the email and we really want to preserve fidelity. 

Additionally, we can't assume any headers are present the way the model currently works. Try something like `From_name=(message.From or '')` which should set them as empty strings if they are `None`.